airflow/contrib/operators/cloudml_operator_utils.py

```diff
@@ -170,10 +186,20 @@ def validate_err_and_count(summary):
     if not callable(validate_fn):
         raise AirflowException("`validate_fn` param must be callable.")
 
+    if dag is not None:
+        default_args = dag.default_args
+        if default_args is not None:
```

Comment:
Maybe you could merge two ifs into one,
if dag is not None and dag.default_args is not None:
  default_args = dag.default_args
  ...